draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -568,7 +568,12 @@ in the Issuer identifier.
 
 # Relationships to Other Documents
 
-TBD
+This specification defines validation and processing rules for verifiable credentials using JSON-only
```

Comment:
`using JSON-only` stood out here for me. `using only JSON payload` or just `using JSON payloads` should be fine? (sure, some can say JSON-LD is also JSON, but JSON-only does not really make that point clearer IMO)

Reply:
We could also add something like:
> Compared to other credential formats, SD-JWT VC was created to be focused on developers who use JSON-based data models and who are familiar with the JOSE stack.

Reply:
@Sakurann's suggestion of just `using JSON payloads` seems good 
```suggestion
This specification defines validation and processing rules for verifiable credentials using JSON
```